keyrod.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.16'
   spec.add_development_dependency 'rake', '~> 12.3'
   spec.add_development_dependency 'rspec', '~> 3.7'
-  spec.add_development_dependency 'rubocop', '~> 0.52'
+  spec.add_development_dependency 'rubocop', '~> 0.53'
```

Comment:
You don't need to update a version in the `gemspec`. Notation `~>` tells bundler to update the gem version if the update changes only the last number after the dot. So in this case if you have `rubocop ~> 0.52` in `gemspec` and you call `bundle install` bundler will install the rubocop with major version `0` and latest minor version. If you want to update gems in your project this way just call `bundle update`.
That said, feel free to leave it on `0.53`, it doesn't really matter much.